client/payment-details/payment-method/ideal/index.js

```diff
@@ -0,0 +1,152 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import PaymentDetailsPaymentMethodDetail from '../detail';
+
+/**
+ * Extracts and formats payment method details from a charge.
+ *
+ * @param {Object} charge The charge object.
+ * @return {Object}       A flat hash of all necessary values.
+ */
+const formatPaymentMethodDetails = ( charge ) => {
+	const { billing_details: billingDetails, payment_method: id } = charge;
+
+	const {
+		bank: bankName,
+		bic,
+		country: countryCode,
+		iban_last4: last4,
+		verified_name: verifiedName,
+	} = charge.payment_method_details.ideal;
+
+	const { name, email, formatted_address: formattedAddress } = billingDetails;
+	// Use the full country name.
+	const country = wcSettings.countries[ countryCode ];
+
+	return {
+		bankName,
+		bic,
+		last4,
+		verifiedName,
+		id,
+		name,
+		email,
+		country,
+		formattedAddress,
```

Comment:
I notice the order is not the same when comparing the placeholder and the formatting:
https://github.com/Automattic/woocommerce-payments/pull/2681/files#diff-b36b04f7de333baf74368547db64a79a13e73caab3b935e329bb674d2862bd0fR59

Does the order matter here?

Reply:
@dwainm  I'm not 100% sure, but I've rearranged it to mimic the loaded result just in case, and also removed an unused value. Thank you! 👍